addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/scanrules/ScanRuleMetadataUnitTest.java

```diff
@@ -74,14 +81,63 @@ void shouldParseMetadataYaml() {
         assertThat(metadata.getCategory(), is(equalTo(Category.INJECTION)));
         assertThat(metadata.getRisk(), is(equalTo(Risk.INFO)));
         assertThat(metadata.getConfidence(), is(equalTo(Confidence.LOW)));
-        assertThat(metadata.getCweId(), is(equalTo(0)));
-        assertThat(metadata.getWascId(), is(equalTo(0)));
+        assertThat(metadata.getCweId(), is(equalTo(123)));
+        assertThat(metadata.getWascId(), is(equalTo(456)));
         assertThat(
                 metadata.getAlertTags(), is(equalTo(Map.of("name1", "value1", "name2", "value2"))));
         assertThat(metadata.getOtherInfo(), is(equalTo("Any other Info")));
         assertThat(metadata.getStatus(), is(equalTo(AddOn.Status.alpha)));
         assertThat(metadata.getCodeLink(), is(equalTo("https://www.example.com/codelink")));
         assertThat(metadata.getHelpLink(), is(equalTo("https://www.example.com/helplink")));
+        assertThat(metadata.getAlertRefOverrides().size(), is(equalTo(2)));
+        assertThat(
+                metadata.getAlertRefOverrides().get("12345-1").getName(),
+                is(equalTo("Alert Ref Override 1")));
+        assertThat(
+                metadata.getAlertRefOverrides().get("12345-2").getName(),
+                is(equalTo("Alert Ref Override 2")));
+    }
+
+    @Test
+    void shouldHandleAllAlertRefOverrideFields() {
+        // Given
+        String yaml =
+                """
+id: 12345
+name: Active Vulnerability Title
+alertRefOverrides:
+  12345-1:
+    name: Alert Ref 1
+    description: Full description
+    solution: The solution
+    references:
+      - Reference 1
+      - Reference 2
```

Comment:
I know these are technically free text, but to make it clear I think we should use. ex: https://one.example.com/ https://two.example.com/

Reply:
This was marked as resolved but it's still using free text.

Reply:
I think there were two places and I resolved it having seen just the first one :wink: